common/djangoapps/util/tests/test_milestones_helpers.py

```diff
@@ -0,0 +1,93 @@
+"""
+Tests for the milestones helpers library, which is the integration point for the edx_milestones API
+"""
+
+from django.test import TestCase
+from mock import patch
+from util import milestones_helpers
+from xmodule.modulestore.tests.factories import CourseFactory
+
+
+class MilestonesHelpersTestCase(TestCase):
+
+    def setUp(self):
+        """
+        Test case scaffolding
+        """
+        self.course = CourseFactory.create(
+            metadata={
+                'entrance_exam_enabled': True,
+            }
+        )
+
+        self.user = {'id': '123'}
+
+        self.milestone = {
+            'name': 'Test Milestone',
+            'namespace': 'doesnt.matter',
+            'description': 'Testing Milestones Helpers Library',
+        }
+
+    @patch.dict('django.conf.settings.FEATURES', {'MILESTONES_APP': False})
```

Comment:
I think it might be the case that you can make this `@patch.dict` call on the class, and it will auto-patch each method.